assets/scss/_custom.scss

```
@@ -96,6 +96,25 @@ body.td-home main[role="main"] > section:first-of-type .content p:first-child {
   }
 }
 
+.section-feature#kubeweekly {
+
+  .kubeweekly-signup {
+    color: $dark-grey;
+    font-weight: 400;
+    font-size: 1.8rem;
+  }
+  > .kubeweekly-inner {
+    form p {
+      text-align: center;
+      font-size: 1.2rem;
+      color: $black;
+    }
+    margin-left: auto;
+    margin-right: auto;
+    padding-bottom: 2rem;
```

```suggestion
    padding-top: 2rem;
```
Making this padding on the top vertically centres the block. Right now a `<be>` is used as a hack to create the gap but the padding is better imo.